src/storage-client/src/types/sources.rs

```diff
@@ -119,7 +122,8 @@ impl<T: Timestamp + Lattice + Codec64> ResumptionFrontierCalculator<T>
             .open(persist_location.clone())
             .await
             .expect("error creating persist client")
-            .open_writer::<SourceData, (), T, Diff>(*remap_shard)
+            // TODO: Any way to plumb the GlobalId to this?
+            .open_writer::<SourceData, (), T, Diff>(*remap_shard, "resumption remap")
```

Comment:
I think, by convention the first `source_export` is the "main source" from which all the source exports derive, so we could use that for the remap shard. I don't know how much we want to rely on that, though. cc @petrosagg